sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1638,6 +1638,58 @@ class Dataset[T] private[sql](
       c5: TypedColumn[T, U5]): Dataset[(U1, U2, U3, U4, U5)] =
     selectUntyped(c1, c2, c3, c4, c5).asInstanceOf[Dataset[(U1, U2, U3, U4, U5)]]
 
+  /**
+   * Checks if the condition is true for any of the rows.
+   * {{{
+   *   peopleDs.forany($"age" > 15)
+   * }}}
+   *
+   * @group typedrel
+   * @since 3.2.0
```

Comment:
This can't be since 3.2.0
This would have to be in R, Scala APIs too
But this doesn't seem worth it - it isn't faster to execute

Reply:
That’s true. So do you recommend we drop this as it would be too much to implement in R and Python?

I believe the code is for Scala API. 
Just FYI, isEmpty method is absent in Python API although present in Scala API. 

Let me know if we still wanna go forward, I can change the version then. If you let me know the right version.

Reply:
Sorry yes I meant Python/R. I don't think this is worthwhile if it isn't optimizing or saving the caller much code

Reply:
No problems :)

i will close the PR